src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,44 @@
+import { Link } from "react-router-dom";
+
+
+function ArticleCard({ searchedArticles }) {
+  return (
+    <section className="article-list">
+      {searchedArticles.map((article) => {
+        return (
+          <section className="article-card" key={article.article_id}>
+            <ul className="cardArticleList">
+              <li className="idLink">
+                <Link to={`/article/${article.article_id}`}>
+                  <h4>{article.title}</h4>
+                </Link>
+              </li>
+              <li className="topicLink">
+                <Link to={`/topic/${article.topic}`}>
+                <p>Topic: {article.topic}</p>
+                </Link>
+              </li>
+              <li>
+                <p>Topic: {article.created_at}</p>
+              </li>
+              <li>
+              <li>
+                <p>{article.body}</p>
+              </li>
+                <p>Posted by {article.author}</p>
+              </li>
+              <li>
+                <p>{article.votes} Votes</p>
+              </li>
+              <li>
+                <p>{article.comment_count} people have commented on this article</p>
+              </li>
+            </ul>
+          </section>
+        );
+      })}
+    </section>
+  );
+}
```

Comment:
ArticleCard and ArticleCardTopic are identical apart from the data you're mapping over, this is the perfect chance to have a reusable component instead of two of the same  - please use one for both the topic list of articles and main list of articles

Reply:
So would I use the same state for both components? Would this mean I would need to move state to app.js so I can access the state in both components?